### PR TITLE
translated several paragraphs in embedding.rst

### PR DESCRIPTION
**Pull request by XD-DENG**
Translated several paragraphs in embedding.rst

This is also my first try on Julia documentation translation. I'm trying to find a way to contribute to julia.

Many thanks

### CODE REVIEW

File: manual/embedding.rst [hunk 1]
```diff
@@ -95,9 +95,11 @@ Its second argument ``args`` is an array of ``jl_value_t*`` arguments and ``narg
 内存管理
 ========================
 
-As we have seen, Julia objects are represented in C as pointers. This raises the question of who is responsible for freeing these objects.
+..As we have seen, Julia objects are represented in C as pointers. This raises the question of who is responsible for freeing these objects.:
+正如我们看到的，Julia的对象在C中是以指针形式呈现的。而这也就给出了一个问题：由谁来负责释放这些对象对应的内存呢？
```
Comment (yuyichao) [inline manual/embedding.rst]: 中英文切换的时候加空格.

**Comment by XD-DENG** [comment]
Added space as you kindly indicated.

This is my first time to commit to this project. Only three paragraphs. Please understand as I'm trying to get familiar with the work flow of this project. Thanks.

(Chinese input method on my laptop is not working well, so I comment in English. Sorry for this)

File: manual/embedding.rst [hunk 2]
```diff
@@ -95,9 +95,11 @@ Its second argument ``args`` is an array of ``jl_value_t*`` arguments and ``narg
 内存管理
 ========================
 
-As we have seen, Julia objects are represented in C as pointers. This raises the question of who is responsible for freeing these objects.
+..As we have seen, Julia objects are represented in C as pointers. This raises the question of who is responsible for freeing these objects.:
+正如我们看到的，Julia 的对象在 C 中是以指针形式呈现的。而这也就给出了一个问题：由谁来负责释放这些对象对应的内存呢？
 
-Typically, Julia objects are freed by a garbage collector (GC), but the GC does not automatically know that we are holding a reference to a Julia value from C. This means the GC can free objects out from under you, rendering pointers invalid.
+..Typically, Julia objects are freed by a garbage collector (GC), but the GC does not automatically know that we are holding a reference to a Julia value from C. This means the GC can free objects out from under you, rendering pointers invalid.:
+一般情况下，Julia 的对象由垃圾回收机制来负责释放内存，但垃圾回收机制并不会自动获知我们在使用一个自 C 向 Julia 的引用值。这意味着垃圾回收机制会释放对象，使得所使用的指针无效。
```
Comment (yuyichao) [inline manual/embedding.rst]: 个人认为有些地方这样通顺点

> 一般情况下，Julia 的对象由垃圾回收机制来释放，但垃圾回收机制并不能自动获知我们正在 C 中使用 Julia 对象的引用。这意味着垃圾回收机制可能会释放我们正在使用的对象，造成该指针失效。

File: manual/embedding.rst [hunk 3]
```diff
@@ -122,7 +124,8 @@ Several Julia values can be pushed at once using the ``JL_GC_PUSH2`` , ``JL_GC_P
 控制垃圾回收
 ---------------------------------------------------
 
-There are some functions to control the GC. In normal use cases, these should not be necessary.
+..There are some functions to control the GC. In normal use cases, these should not be necessary.:
+有一些函数可以帮助控制垃圾回收。但在一般情况下，它们都不需被专门使用。
```
Comment (yuyichao) [inline manual/embedding.rst]: > 在一般情况下，它们都不需要被用到。

Comment (XD-DENG) [inline manual/embedding.rst]: Addressed as you suggested.

Next time I'll translate more contents for one pull request.

**Comment by yuyichao** [comment]
LGTM. I'll give others some time to comment and will merge a few hours later otherwise.

**Comment by GaZ3ll3** [comment]
good to merge.

**Comment by XD-DENG** [comment]
@GaZ3ll3 Hi,  thanks for suggesting. I'm not so familiar with the rebase and squash feature. May you merge it firstly? and I'll learn about these features later and apply next time.
Fully understand that we need rules on committing to ensure that whole project goes well.

**Comment by yuyichao** [comment]
Rebased manually and pushed.

**Comment by yuyichao** [comment]
@XD-DENG See https://help.github.com/articles/about-git-rebase/ and the related links for rebase and squash.

**Comment by XD-DENG** [comment]
@yuyichao Many thanks.

**Pull request closed without merging**